src/windows.rs

```diff
@@ -362,7 +362,7 @@ impl WinCredential {
                 username: user.to_string(),
                 target_name: target.to_string(),
                 target_alias: String::new(),
-                comment: format!("{user}@{service}:{target} (keyring v{VERSION})"),
+                comment: "keyring v{VERSION}".to_string(),
```

Comment:
Looks like you need the format! statement to convert the {VERSION}.

Reply:
Ah! Another lesson for me to review my patches with a full night's rest :-) I've pushed a fix.